docs/cache/getting-started.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 3
+sidebar_label: Getting Started
+title: Getting Started
+pagination_prev: null
+pagination_next: null
+description: Jump in and get started quickly using Momento Cache with your SDK of choice.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started with Momento Cache
+
+If you're looking to jump in and get started with Momento Cache with an SDK, you've come to the right place.
+
+If you want to quickly get a feel for the Momento Cache experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser).
+
+## Using the Momento Console
+
+Follow our [step-by-step instructions](./develop/authentication/api-keys) to create an API key via the [Momento console](https://console.gomomento.com). Come back here when you have your key.
```

Comment:
This seems like a prerequisite step that should be called out before any of the "Using" sections